bangazon.py

```diff
@@ -102,7 +102,16 @@ def select_customer(self):
 
   def create_payment_type(self):
     self.page_clear()
-    print('option 3 - Create Payment Type')
+    print('Create your payment type.')
+    time.sleep(.5)
+    print('Enter your payment name.')
+    payment_name = input('Payment Name: ')
+    print('Enter your payment account number.')
+    payment_accountNum = input('Payment Account #: ')
+    new_payment = Payment(payment_name, payment_accountNum)
```

Comment:
below this line
self.serialize_data(self.all_customers, 'customers.p')
    time.sleep(1)